index.html

```diff
@@ -51,5 +51,8 @@
       addCampaign("https://github.com/Yamaino/BGS-stacksofmiddenland")
       addCampaign("https://github.com/wchomik/BGS-testcampaing");
     </script>
+    <script>
+      addCampaign("https://github.com/Yamaino/BGS-stacksofmiddenland");
```

Comment:
Why is it here? This campaing is already added.

Reply:
I think that I totally forgot to remove it. I have added this in me repository some time ago, but never pushed it.